lib/cube/buildCubeShape/Cube.js

```diff
@@ -49,15 +49,18 @@ class Cube {
     clownface({ dataset, term: this.observationSet })
       .addOut(ns.rdf.type, ns.cube.ObservationSet)
 
-    clownface({ dataset, term: this.shape })
+    const shapeDataset = rdf.dataset()
+
+    clownface({ dataset: shapeDataset, term: this.shape })
```

Comment:
I think the only real improvement could be to get rid of the `setGraph` function and `shapeDataset` if you initialise the pointer with a named graph

```suggestion
    clownface({ dataset, term: this.shape, graph: shapeGraph })
```

Reply:
But later one adds datasets produced by the dimensions, 

```js
    for (const dimension of this.dimensions.values()) {
      shapeDataset.addAll(dimension.toDataset({ shape: this.shape }))
    }
    ```
    
    Apparently that part does not respect the graphName used in the clownface object (shapeDataset)

Reply:
Uh, ok, I have no more comments then. Let's keep it as is it